src/modules/services_manager_service.rb

```diff
@@ -86,38 +90,40 @@ def load_units
           service.chomp! SERVICE_SUFFIX
           units[service] = {
             :status => status,
-            # bsc#956043 service can be 'just being activated' or 'reloaded'
-            # See https://github.com/systemd/systemd/blob/7152869f0a4a4612022244064cc2b3905b1e3fc7/src/basic/unit-name.c#L844
-            :active => (
-              active == Status::ACTIVE ||
-              active == Status::ACTIVATING ||
-              active == Status::RELOADING
-            ),
             :description => description.join(' ')
           }
         end
       end
 
-      def extract_services_from_unit_files
-        supported_unit_files.each do |name, status|
-          next if services[name]
+      def extract_services
```

Comment:
I would rather prefer having two separates methods as we did before (`extract_services_from_unit_files` and `extract_services_from_units`). But I see your point and maybe you could consider just reducing the current method to two lines:

```ruby
def extract_services
  extract_services_from_units
  extract_services_from_unit_files
end
```

In that case, we would keep the `extract_services_*` methods short and you still only need to call 1 method.